doc-2581: clarify stylus (connected mode) (#9906)
spectrocloud/librarium: docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md

:::

- Depending on the deployment method used, there are some [network failure considerations](../edgeforge-workflow.md) to
  take into account:
  - For centrally managed Edge cluster if a required pack cannot be downloaded, the cluster processing fails. There is a

>     periodic reconcile attempt to download the missing packs.

Reviewer: ```suggestion
  - For a centrally managed Edge cluster, if a required pack cannot be downloaded, the cluster processing fails. There is a periodic reconciliation attempt to download the missing packs.
```

  - For locally managed Edge cluster with internet or local network resources, if a required pack is not available in

>     the content bundle, it cannot be downloaded from the internet or local network accessible registry.

Reviewer: ```suggestion
  - For a locally managed Edge cluster with internet or local network resources, if a required pack is not available in the content bundle, it cannot be downloaded from the internet or a local network accessible registry.
```

  - For locally managed Edge cluster with no internet or local network resources, if a required pack is not available in

>     the content bundle, it cannot be downloaded from the internet or local network accessible registry.

Reviewer: ```suggestion
  - For a locally managed Edge cluster with no internet or local network resources, if a required pack is not available in the content bundle, it cannot be downloaded from the internet or a local network accessible registry.
```
